BlueprintUI/Sources/Layout/Builder/EqualStackChild.swift

```diff
@@ -0,0 +1,17 @@
+/// Wraps around an Element for consistency with `StackChild` and `GridRowChild`.
+/// In the future this struct could hold traits used for laying out inside an EqualStack
+extension EqualStack {
+    public struct Child {
+        public let element: Element
+
+        public init(element: Element) {
+            self.element = element
+        }
+    }
+}
+
+extension EqualStack.Child: ElementBuilderChild {
+    public init(_ element: Element) {
```

Comment:
This doesn't seem super useful / I'd just make the `EqualStack.Child` implement this, and make it conform to this directly.

Reply:
Agreed.